utils/threshAlgos.py

```diff
@@ -433,20 +433,20 @@ def anaUltraThreshold(args,thrFilename,GEBtype="short",outputDir=None,fileScurve
     # Do we return analyzed TTree?
     from multiprocessing import current_process
     if (current_process().name == 'MainProcess'):
-        return 
+        return
```

Comment:
@bdorney c'est quoi ce truc?

Reply:
mmhmm seems my vimrc isn't doing what I thought it was...I thought it should be stripping trailing white spaces automatically...I guess this is *not* the case :D

Reply:
Nah, I meant this and the next line (double `return` statements)

Reply:
the more `return` statements you have the faster you get your output right? right!?

Reply:
(indeed the `return thrAnaTree` is what is desired here)

Reply:
still needs to be resolved.

Reply:
Indeed, extra return has to be removed!